src/components/DatetimePicker/DatetimePicker.vue

```diff
@@ -142,11 +144,17 @@ export default {
 		// eslint-disable-next-line
 		value: {
```

Comment:
So let's add typing then?
If we're enforcing a string?

Reply:
I was assuming that the typing was intentional. Okay then, I'll change the typing instead.